packages/cli/yaml/docs-validator/src/docsAst/visitDocsConfigFileYamlAst.ts

```diff
@@ -178,6 +178,7 @@ export async function visitDocsConfigFileYamlAst({
                 context
             });
         },
+        products: noop,
```

Comment:
https://github.com/fern-api/fern/blob/fb89f62fc49f85319a94e3fd33ca04f96f4ccf3b/packages/cli/yaml/docs-validator/src/docsAst/visitDocsConfigFileYamlAst.ts#L224-L263

we should validate the keys inside the products file (just like we do versions!) for example, if a file does not exist, then we would want to give a user a heads up